db/default_messages/upcoming_checkout_email

```diff
@@ -0,0 +1,4 @@
+  Hey there, you have equipment reserved today! Please checkout the following items today:\n\n\
```

Comment:
Do we know where those `\r` characters or how to get rid of them? I thought that Git generally handled line endings, so I'm surprised it's there, particularly since these files were generated by @squidgetx (who runs Linux). Any ideas?